plugin.xml

```diff
@@ -17,7 +17,8 @@
     <platform name="ios">
       <header-file src="src/ios/Intercom.h" />
       <source-file src="src/ios/libIntercom.a" framework="true" compiler-flags="-ObjC" />
```

Comment:
Just curious, why is it that you are using `libIntercom.a` here instead of the binary from `Intercom.framework`?